meilisearch-release-changelog/v1.4.0.md

```diff
@@ -63,14 +63,24 @@ This feature can be used together with the `stopWords` and `synonyms` index sett
 Done by @ManyTheFish in #3946
 
 ## Other improvements
-TBD
+
+Meilisearch better hides your data and no longer shows hidden document's fields in error messages. In case you are trying to sort by using a field that is not sortable but there are some of them that are not displayed either, you will see a message like the following:
+
+```
+Available sortable attributes are: price, stock, <..hidden-attributes>.
+```
+
+Done by @vivek-26 in #3945
 
 # Fixes 🐞
 
-* Fix synonyms display (#3946) @ManyTheFish
+  * Fix synonyms display (#3946) @ManyTheFish
+  * Accept the `null` JSON value as a value of the `_vectors` field (#3998) @Kerollmops
+  * Fix the ranking rule by temporarily disabling an assert in the bucket sort algorithm (#4013) @Kerollmops
 
 # Misc
 
+  * Bump charabia to 0.8.3 (#4008) @Kerollmops
```

Comment:
@Kerollmops same looks like it's not a PR for v1.4.0 can you confirm

Reply:
Nope, it has been done in v1.3.2. I don't know what I did there.